### ISSUE TITLE
Failed to activate the atom-limelight package

### ISSUE THREAD

**Issue by rsbatacjr** [germtb/atom-limelight#2]
[Enter steps to reproduce:]

1. Open Atom
2. Open Project that has file update previously.
    There will be popup saying "Previous automatically-saved project state detected"
3. Choose Open in new window and recover step

**Atom**: 1.22.0 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.1
**Thrown From**: [atom-limelight](https://github.com/germtb/atom-limelight) package 0.1.2

### Stack Trace

Failed to activate the atom-limelight package

```
At editor.languageMode.rowRangeForParagraphAtBufferRow is not a function

TypeError: editor.languageMode.rowRangeForParagraphAtBufferRow is not a function
    at markRow (/packages/atom-limelight/lib/atom-limelight.js:31:37)
    at /packages/atom-limelight/lib/atom-limelight.js:65:25
    at Array.forEach (native)
    at /packages/atom-limelight/lib/atom-limelight.js:65:12
    at Object.turnOn (/packages/atom-limelight/lib/atom-limelight.js:110:3)
    at /packages/atom-limelight/lib/atom-limelight.js:91:10
    at Config.module.exports.Config.observeKeyPath (/Applications/Atom.app/Contents/Resources/app/src/config.js:674:13)
    at Config.module.exports.Config.observe (/Applications/Atom.app/Contents/Resources/app/src/config.js:155:27)
    at Object.activate (/packages/atom-limelight/lib/atom-limelight.js:89:15)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:253:25)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:225:38
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:99:21)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:218:32
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:215:40)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:690:42)
    at config.transactAsync (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:665:36)
    at Config.module.exports.Config.transactAsync (/Applications/Atom.app/Contents/Resources/app/src/config.js:367:24)
    at PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:663:23)
    at PackageManager.activate (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:646:50)
    at /Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:854:36
```

### Commands

```
```

### Non-Core Packages

```
atom-limelight 0.1.2 
atom-minify 0.8.0 
goto-definition 1.3.4 
line-navigation 0.1.6 
platformio-ide-terminal 2.7.0 
todo-show 2.1.0 
```

**Comment by lk22** [comment]
i found a similar error with another package but had same line of code 

https://github.com/aki77/atom-expand-region/issues/29

what i did, i changed it from editor.languageMode.rowRangeForParagraphAtBufferRow() to
editor.rowRangeForParagraphAtBufferRow i guess is an change in the editor API at some sort

**Comment by rsbatacjr** [comment]
Thanks, in my case, the error suddenly stop showing. I will close this issue and will post anything if it went back.

**Comment by germtb** [comment]
I am seeing this issue again... I think it's just atom api changing. Should be fixed by v0.1.3
